lint.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+line='----------------------------------------'
+
+for file in `ls *.json && ls */*.json`; do
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏

```suggestion
shopt -s globstar
for file in **/*.json; do
```

Reply:
sweet 🤩

Reply:
Or maybe simply use git, since it knows which json files are tracked:
```suggestion
for file in `git ls-files '*.json'`; do
```

Reply:
the first one is closer to what I was going for, I went with it